clients/wallet/src/stellar_wallet.rs

```diff
@@ -337,6 +337,12 @@ impl StellarWallet {
 		request_id: [u8; 32],
 		stroop_fee_per_operation: u32,
 	) -> Result<TransactionResponse, Error> {
+		// user must not send to self
+		#[cfg(not(feature = "testing-utils"))]
```

Comment:
right now the vault integration tests contains lots of self payment. 
This cfg guard is for avoiding CI issues, until the tests are updated.

Reply:
updated: https://github.com/pendulum-chain/spacewalk/pull/385/commits/d3d2cc1e472c127bcc74cde62d220a34be6a5147